panel.py

```diff
@@ -308,6 +308,38 @@ def __init__(self, name: str, fdt: Fdt2, node: int) -> None:
 		self.lp11_init = fdt.getprop_or_none(node, 'qcom,mdss-dsi-lp11-init') is not None
 		self.init_delay = fdt.getprop_uint32(node, 'qcom,mdss-dsi-init-delay-us')
 
+		# Display Stream Compression
+		self.compression_mode = fdt.getprop_or_none(mode_node, 'qcom,compression-mode')
+		self.has_dsc = self.compression_mode is not None and self.compression_mode.as_str() == "dsc"
```

Comment:
Can you define an enum `CompressionMode` for this similar to `BacklightControl`? Then do `p.compression_mode == CompressionMode.DSC` instead of `p.has_dsc`.

`Panel` is supposed to be a (fairly poor) abstraction of the DT so that in theory one could write a parser for the XML in the ACPI tables for example. So the `self.compression_mode` shouldn't get a raw DT property assigned.